docs/content/get-started/open-project.md

```diff
@@ -23,9 +27,11 @@ There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor regarding
 1. Open the **Navigation** window (Unity Editor menu: **Windows** > **AI** > **Navigation**).
 1. Select the **Bake** tab, and then the **Bake** button.
 
-You can verify that the NavMesh has been baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator**, and checking that Unity displays the correct icon.
+You can verify that the NavMesh has been baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator**, and checking that Unity shows a Navmesh object (in most cases the logo will update to the below).
```

Comment:
```suggestion
You can verify that the NavMesh is baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator** and checking that there's a NavMesh object in the **Hierarchy** window. (The NavMesh object is usually represented by the icon below.)
```

A few bits and bobs here. The phrase "the below" is not so good English.  Also, it's worth saying where you can see the NavMesh object.

Reply:
Note to self - added to new PR.